xdcs-agent/src/main/python/xdcs/cmd/task_reporting.py

```diff
@@ -1,35 +1,35 @@
-import logging
-
 from xdcs.app import xdcs
 from xdcs.cmd import Command
+from xdcs.log_handling import LogHandler, LogLevel
 from xdcs_api.agent_execution_pb2 import TaskId
 from xdcs_api.agent_execution_pb2_grpc import TaskReportingStub
 
-logger = logging.getLogger(__name__)
-
 
 class ReportTaskCompletionCmd(Command):
     _task_id: str
+    _log_handler: LogHandler
 
-    def __init__(self, task_id: str) -> None:
+    def __init__(self, task_id: str, log_handler: LogHandler) -> None:
         self._task_id = task_id
+        self._log_handler = log_handler
 
     def execute(self):
-        logger.info('Reporting task completion: ' + self._task_id)
+        self._log_handler.internal_log("Reporting task completion: " + self._task_id)
 
         stub, task_id = _prepare_reporting_data(self._task_id)
         stub.ReportCompletion(task_id)
 
 
 class ReportTaskFailureCmd(Command):
     _task_id: str
+    _log_handler: LogHandler
 
-    def __init__(self, task_id: str) -> None:
+    def __init__(self, task_id: str, log_handler: LogHandler) -> None:
         self._task_id = task_id
+        self._log_handler = log_handler
 
     def execute(self):
-        logger.info('Reporting task failure: ' + self._task_id)
-
+        self._log_handler.internal_log("Reporting task failure: " + self._task_id, LogLevel.ERROR)
```

Comment:
log z informacją że raportujemy powinien być INFO a nie ERROR bo to nie error

Reply:
ok